README.MD

```diff
@@ -814,7 +814,7 @@ Excludes this endpoint from the generated OpenAPI Specification document.
   }
 ```
 
-It can also be set at the controller level to exclude all of its endpoints from the OpenAPI Specification document.
+Use on controllers to exclude all of its endpoints from the generated OpenAPI Specification document.
```

Comment:
I tweaked the language here so the sentences for method + controller + query all have parallel construction